Layouts/layout-indicatorsDetails-Account.json

```diff
@@ -1,188 +1,297 @@
-              },
-              {
-                "index": 4,
-                "endCol": 2,
-                "height": 24,
-                "startCol": 0,
-                "fieldId": "id",
-                "id": "de9eef00-ae08-11e9-8024-75e863ac67b1"
-              },
-              {
-                "index": 5,
-                "endCol": 2,
-                "height": 24,
-                "startCol": 0,
-                "fieldId": "organizationalunitou",
-                "id": "f51e0ef0-ae08-11e9-8024-75e863ac67b1"
-              },
-              {
-                "index": 6,
-                "endCol": 2,
-                "height": 24,
-                "startCol": 0,
-                "fieldId": "indicatortype",
-                "id": "indicator-type"
-              },
-              {
-                "index": 7,
-                "endCol": 2,
-                "height": 48,
-                "startCol": 0,
-                "fieldId": "groups",
-                "id": "d7132bc0-ae08-11e9-8024-75e863ac67b1"
-              },
-              {
-                "index": 8,
-                "endCol": 2,
-                "dropEffect": "move",
-                "height": 24,
-                "listId": "e2c8c970-a09d-11e9-8956-390f602b039a",
-                "startCol": 0,
-                "fieldId": "emailaddress",
-                "id": "cdc51f10-ae08-11e9-8024-75e863ac67b1"
-              },
-              {
-                "index": 9,
-                "endCol": 2,
-                "dropEffect": "move",
-                "height": 24,
-                "listId": "e2c8c970-a09d-11e9-8956-390f602b039a",
-                "startCol": 0,
-                "fieldId": "internal",
-                "id": "e4328bc0-ae08-11e9-8024-75e863ac67b1"
-              },
-              {
-                "index": 10,
-                "endCol": 2,
-                "dropEffect": "move",
-                "height": 48,
-                "listId": "e2c8c970-a09d-11e9-8956-390f602b039a",
-                "startCol": 0,
-                "fieldId": "comment",
-                "id": "indicator-comment"
-              }
-            ],
-            "moved": false,
-            "displayType": "ROW",
-            "static": false,
-            "w": 1,
-            "h": 3,
-            "y": 0,
-            "x": 0
-          }
-        ],
-        "id": "default-main",
-        "name": "Info"
-      }
-    ],
-    "modified": "2019-07-24T11:49:46.253775898Z",
-    "TypeName": "",
-    "version": -1,
-    "id": "accountRep"
-  },
-  "TypeName": "Account",
-  "version": -1,
-  "fromVersion": "5.0.0"
+	"TypeName": "accountRep",
+	"kind": "indicatorsDetails",
+	"layout": {
+		"TypeName": "",
+		"commitMessage": "",
+		"id": "accountRep",
+		"kind": "indicatorsDetails",
+		"modified": "2020-02-13T16:20:38.30086+02:00",
+		"name": "",
+		"prevKind": "indicatorsDetails",
+		"prevTypeId": "accountRep",
+		"shouldCommit": false,
+		"sortValues": null,
+		"system": false,
+		"tabs": [
```

Comment:
It looks like this file is using tabs... and may be misformatted towards the bottom? It's hard to say - but please take a look.

Reply:
it seems this is something in how git parses this file... locally it looks fine